CometServer/Utilities/StaticLinkedList.cpp

```diff
@@ -53,6 +57,8 @@ namespace utils
 		//Init outlist.
 		elements[outstart].nextindex = outstart + 2;
 		elements[outend].previndex = outend - 2;
+		elements[outstart + 2].previndex = outstart;
+		elements[outend - 2].nextindex = outend;
```

Comment:
Yeah, really, time to write tests for this. Why did we switched the positions of `instart`/`outstart` and `inend`/`outend` to begin with? I vaguely remembering they had to be switched for some reason, but now it seems to be simpler the old way (at least these additions weren't necessary back then).